public class exception2
{
	public static void main(String[] args) 
	{
		try
		{
			int data=50/0;  
		}
		catch(ArithmeticException e) 	
		{
			System.out.println("excepgsdhfkejgsjg"); 
			System.out.println(e); 	
		}
		System.out.println("hi");
		System.out.println("hello");
		System.out.println("rest of the code");
	}
}
